contrib/local-environment/keycloak/init-keycloak.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# init-keycloak.sh
+kcadm_opts="--no-config --server http://localhost:8080/auth --realm master --user admin@example.com --password password -r master"
+res="$(docker-compose -f docker-compose-keycloak.yaml exec keycloak /opt/jboss/keycloak/bin/kcadm.sh get clients/0493c7c6-6e20-49ea-9acb-627c0b52d400 --fields clientId $kcadm_opts)"
+
+if [[ "$res" == *"Resource not found for url"* ]]; then
```

Comment:
Double square brackets isn't standard `sh` so your script isn't guaranteed to be compatible (it doesn't work by default on my ubuntu 18.04)

Reply:
Got rid of all scripts to prepare keycloak and rely on keycloak import dir.